test/images/image-util.sh

```diff
@@ -130,8 +130,8 @@ push() {
   # The manifest command is still experimental as of Docker 18.09.2
   export DOCKER_CLI_EXPERIMENTAL="enabled"
   # Make archs list into image manifest. Eg: 'amd64 ppc64le' to '${REGISTRY}/${IMAGE}-amd64:${TAG} ${REGISTRY}/${IMAGE}-ppc64le:${TAG}'
-  manifest=$(echo "$archs" | ${SED} -e "s~[^ ]*~$REGISTRY\/$IMAGE\-&:$TAG~g")
-  docker manifest create --amend "${REGISTRY}/${IMAGE}:${TAG}" ${manifest}
+  while IFS='' read -r line; do manifest+=("$line"); done < <(echo "$archs" | ${SED} -e "s~[^ ]*~$REGISTRY\/$IMAGE\-&:$TAG~g")
+  docker manifest create --amend "${REGISTRY}/${IMAGE}:${TAG}" "${manifest[@]}"
```

Comment:
```
In ./test/images/image-util.sh line 134:
  docker manifest create --amend "${REGISTRY}/${IMAGE}:${TAG}" ${manifest}
                                                               ^---------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
I cannot try it completely. but I have tested a part of this. In my test, `manifest` has the same value as before the change.

- before the change (output of bash -x)
```
$ sudo make all-push WHAT=redis
...
+++ sed -e 's~[^ ]*~gcr.io/kubernetes-e2e-test-images\/redis\-&:1.0~g'
++ manifest='gcr.io/kubernetes-e2e-test-images/redis-amd64:1.0
gcr.io/kubernetes-e2e-test-images/redis-arm:1.0
gcr.io/kubernetes-e2e-test-images/redis-arm64:1.0
gcr.io/kubernetes-e2e-test-images/redis-ppc64le:1.0
gcr.io/kubernetes-e2e-test-images/redis-s390x:1.0'
++ docker manifest create --amend gcr.io/kubernetes-e2e-test-images/redis:1.0 gcr.io/kubernetes-e2e-test-images/redis-amd64:1.0 gcr.io/kubernetes-e2e-test-images/redis-arm:1.0 gcr.io/kubernetes-e2e-test-images/redis-arm64:1.0 gcr.io/kubernetes-e2e-test-images/redis-ppc64le:1.0 gcr.io/kubernetes-e2e-test-images/redis-s390x:1.0
```

- after
```
$ sudo make all-push WHAT=redis
...
+++ sed -e 's~[^ ]*~gcr.io/kubernetes-e2e-test-images\/redis\-&:1.0~g'
++ manifest+=("$line")
++ IFS=
++ read -r line
++ manifest+=("$line")
++ IFS=
++ read -r line
++ manifest+=("$line")
++ IFS=
++ read -r line
++ manifest+=("$line")
++ IFS=
++ read -r line
++ manifest+=("$line")
++ IFS=
++ read -r line
++ docker manifest create --amend gcr.io/kubernetes-e2e-test-images/redis:1.0 gcr.io/kubernetes-e2e-test-images/redis-amd64:1.0 gcr.io/kubernetes-e2e-test-images/redis-arm:1.0 gcr.io/kubernetes-e2e-test-images/redis-arm64:1.0 gcr.io/kubernetes-e2e-test-images/redis-ppc64le:1.0 gcr.io/kubernetes-e2e-test-images/redis-s390x:1.0
```